client/signup/steps/import-url/index.jsx

```diff
@@ -5,162 +5,211 @@
 import React, { Component } from 'react';
 import { localize } from 'i18n-calypso';
 import { connect } from 'react-redux';
-import { flow, get, indexOf, inRange } from 'lodash';
-import { isWebUri } from 'valid-url';
-import { parse as parseURL } from 'url';
+import { flow, isEqual } from 'lodash';
 
 /**
  * Internal dependencies
  */
+import Card from 'components/card';
+import Notice from 'components/notice';
 import StepWrapper from 'signup/step-wrapper';
 import SignupActions from 'lib/signup/actions';
-import FormTextInputWithAction from 'components/forms/form-text-input-with-action';
-import { setNuxUrlInputValue, setValidationMessage } from 'state/importer-nux/actions';
+import FormTextInput from 'components/forms/form-text-input';
+import FormButton from 'components/forms/form-button';
+import { fetchIsSiteImportable, setNuxUrlInputValue } from 'state/importer-nux/actions';
 import FormInputValidation from 'components/forms/form-input-validation';
-import FormSettingExplanation from 'components/forms/form-setting-explanation';
 import {
+	getNuxUrlError,
 	getNuxUrlInputValue,
 	getSiteDetails,
-	getUrlInputValidationMessage,
 	isUrlInputDisabled,
 } from 'state/importer-nux/temp-selectors';
+import { validateImportUrl } from '../../../my-sites/importer/site-importer/url-validation';
 
-const normalizeUrlForImportSource = url => {
-	// @TODO sanitize? Prepend https:// ..?
-	return url;
-};
+// const normalizeUrlForImportSource = url => {
```

Comment:
Lets chop this out, if necessary, before we merge 👍

Reply:
Will do.